### PR TITLE
fix: display a supported wordcloud for IE 6-11

### PR DESCRIPTION
**Pull request by pascalhuynh**

### CODE REVIEW

File: assembl/static2/js/app/utils/globalFunctions.js [hunk 1]
```diff
@@ -381,4 +381,8 @@ export const formatedTagList = (tags: Tags): Array<TagProps> =>
       }
       return result;
     }, [])
-    : []);
\ No newline at end of file
+    : []);
+
+// Function that checks wheter the browser in IE from v6 to v11
+// eslint-disable-next-line
+export const isIE = () => /*@cc_on!@*/ false || !!document.documentMode;
```
Comment (ayyazdaniaryan) [inline assembl/static2/js/app/utils/globalFunctions.js]: Is this the best way? Common sense says use the `useragent`, and seems like `Trident` is the domain of IE 11 and below. https://stackoverflow.com/a/22242528

Comment (pascalhuynh) [reply assembl/static2/js/app/utils/globalFunctions.js]: I got the info from here: https://stackoverflow.com/questions/9847580/how-to-detect-safari-chrome-ie-firefox-and-opera-browser. It got 1463 upvotes...

Comment (ayyazdaniaryan) [reply assembl/static2/js/app/utils/globalFunctions.js]: Alright, looks like I was wrong. Useragent can be spoofed. Vas y

**Comment by pascalhuynh** [comment]
@ayyazdaniaryan just updated the branch. Thanks for the review, ill update develop right after your approval.

**Comment by pascalhuynh** [comment]
Thanks @ayyazdaniaryan will merge after travis check

**Pull request merged**